WPMediaPicker.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WPMediaPicker'
-  s.version       = '1.8.5'
+  s.version       = '1.8.6'
```

Comment:
Please update the podspec version to be a beta.
```suggestion
  s.version       = '1.8.6-beta.1'
```

Reply:
Resolved here https://github.com/wordpress-mobile/MediaPicker-iOS/pull/399/commits/8ff84d4984635d700219e0129c5c1e39dc2c2024 😄

Reply:
Given that `1.8.6` has [recently been released](https://github.com/wordpress-mobile/MediaPicker-iOS/commit/8f261918313dd551cc7394d78d0574b458064579), I think this should now be `1.8.7-beta.1`

Reply:
Resolved here https://github.com/wordpress-mobile/MediaPicker-iOS/pull/399/commits/ddf67174b0aeaf8db5716194e27767a2f0613d34 😄